test/correctness/mul_div_mod.cpp

```diff
@@ -574,16 +575,23 @@ int main(int argc, char **argv) {
 
     Halide::Internal::ThreadPool<bool> pool;
     std::vector<std::future<bool>> futures;
+    /*
     for (int vector_width : mul_vector_widths) {
         std::cout << "Testing mul vector_width: " << vector_width << "\n";
         auto f = pool.async(test_mul, vector_width, scheduling, target);
         futures.push_back(std::move(f));
     }
+    */
 
     for (int vector_width : div_vector_widths) {
         std::cout << "Testing div_mod vector_width: " << vector_width << "\n";
-        auto f = pool.async(test_div_mod, vector_width, scheduling, target);
-        futures.push_back(std::move(f));
+        /*
```

Comment:
Ditto-- stale comments?

Reply:
This is all based on a WIP branch of mine, so there's a bunch of stale temporary stuff. I need to get it into a better state.

Reply:
Done.